doc/source/user_guide/index.rst

```diff
@@ -0,0 +1,53 @@
+.. _ref_user_guide:
+
+User guide
+##########
+
+Granta MI record lists
+----------------------
+
+Record lists are collections of records, and can be used in any situation where it is useful to have
+a persistent, server-side list of records. Records are explicitly added to a list by a user, and so
+do not need to have anything in common (as would be typical for a set of search results), and do not
+necessarily need be in the same table or database. The user has complete flexibility in deciding
+which records to add to a list.
+
+The record list management capability is available to users in the following front-end tools:
+
+* The One MI Favorites app, as 'Favorites lists'
+* The MI Materials Gateway, as 'Favorites lists'
+* This package
+
+However, since these front-ends all share a common implementation in Granta MI, record lists can be
+created, accessed, and modified in any front-end with full interoperability. The documentation in
+this package is only concerned with this package, for more information about the other
+tools mentioned in the preceding list consult the :MI_docs:`Granta MI Favorites <One_MI_Favorites>`
+and :MI_docs:`Granta MI Materials Gateway for Ansys Workbench <Granta_MI_MG_Help_AWB>`
+documentation. Documentation is available for other Gateway applications within the help site.
+
+
+Record list operations
+----------------------
+
+This package provides access to the following record list operations:
+
+* Record list management, including creating, deleting, and modifying record lists
+* Modifying the contents of a record list
+* Searching for record lists based on their properties and contents
+* Managing the record list lifecycle, including publishing, revising, and un-publishing record lists
+* Subscribing and unsubscribing the current user to published record lists
+
+Note that only the current user can be subscribed and unsubscribed to record lists. It is currently
+not possible to subscribe users to a list other than the user used to establish the connection to
```

Comment:
"To subscribe other users to a list"? I'm not sure what this is intended to mean

Reply:
I was trying to say something like "you can only subscribe yourself to a list, not other people" - one requested feature has been that if you're an admin, you could bulk subscribe a list of 200 people to a list, so they have it by default.

Reply:
Is it enough to just say "Note that only the current user can be subscribed and unsubscribed to record lists."?

Reply:
Yes, I believe it probably is. I'll update.